python_scripts/ensemble_gradient_boosting.py

```diff
@@ -265,7 +259,9 @@ def generate_data(n_samples=50):
 # second tree corrects the first tree's error, while the third tree corrects the
 # second tree's error and so on).
 #
-# We will compare the generalization performance of random-forest and gradient
+# ## First comparison of GBDT vs random forests
```

Comment:
```suggestion
# ## First comparison of GBDT vs. random forests
```

Reply:
I'm wondering why "First" here. I would drop it since you might refer to another document.